.github/actions/detect-workflow/main_test.go

```diff
@@ -16,6 +16,8 @@ package main
 
 import (
 	"context"
+	"github.com/google/go-cmp/cmp"
```

Comment:
This should go in the lower block of imports below the stdlib imports
Usually it's something like

```
import (
  <stdlib packages>

  <3rd party packages>

  <packages from this project/repo>
)
```